plugin/vcs/github/github_test.go

```diff
@@ -91,6 +91,71 @@ func TestProvider_FetchUserInfo(t *testing.T) {
 	assert.Equal(t, want, got)
 }
 
+func TestProvider_FetchCommitByID(t *testing.T) {
+	p := newProvider(
+		vcs.ProviderConfig{
+			Client: &http.Client{
+				Transport: &common.MockRoundTripper{
+					MockRoundTrip: func(r *http.Request) (*http.Response, error) {
+						assert.Equal(t, "/repos/octocat/Hello-World/git/commits/7638417db6d59f3c431d3e1f261cc637155684cd", r.URL.Path)
+						return &http.Response{
+							StatusCode: http.StatusOK,
+							// Example response taken from https://docs.github.com/en/rest/git/commits#get-a-commit
+							Body: io.NopCloser(strings.NewReader(`
+{
+  "sha": "7638417db6d59f3c431d3e1f261cc637155684cd",
+  "node_id": "MDY6Q29tbWl0NmRjYjA5YjViNTc4NzVmMzM0ZjYxYWViZWQ2OTVlMmU0MTkzZGI1ZQ==",
+  "url": "https://api.github.com/repos/octocat/Hello-World/git/commits/7638417db6d59f3c431d3e1f261cc637155684cd",
+  "html_url": "https://github.com/octocat/Hello-World/commit/7638417db6d59f3c431d3e1f261cc637155684cd",
+  "author": {
+    "date": "2014-11-07T22:01:45Z",
```

Comment:
What's the timezone for this timestamp? Maybe we should comment about it in the API message, and be careful about timestamp conversion. Internally, we should use UTC/Unix time consistently.

Reply:
According to RFC3339, no timezone implies UTC: https://datatracker.ietf.org/doc/html/rfc3339#section-5.8

Also this is what GitHub would response to API clients, not in our control.

Reply:
Yeah, we just need to leave a comment on the API message?

Reply:
Could you make a direct suggestion on the PR? I'm not which place exactly you're referring to. Though I'd doubt the worthiness of documenting such, as RFC3339 format is timezone aware by design.

Reply:
Added in https://github.com/bytebase/bytebase/pull/1417/commits/bc1ac582602936a4924047f1d1609fcdf52011ba.